src/main/java/org/wojciechklups/google/SheetsService.java

```diff
@@ -22,10 +25,31 @@
  * @author Author: wklups
  * @timestamp Date: 2022-08-29 15:14:21 +0200 (29 sie 2022)
  */
+@Service
 public class SheetsService
 {
+    @Value("${sheet.name}")
+    private String sheetName;
+
+    @Value("${sheet.first}")
+    private Boolean isFirstSheetFlag;
+
+    private static String SHEET_NAME;
+
+    private static Boolean IS_FIRST_SHEET_FLAG;
```

Comment:
Btw why we need boolean? can't we use sheet name? even you set this boolean in yml so what's the point?